simplq/src/styles/footer.module.scss

```diff
@@ -53,12 +53,19 @@
         & > ul {
           margin: 0;
           padding: 0;
+          display: inline-flex;
           list-style-type: none;
-
+          li + li{
+            padding-left:1rem;
+          }
```

Comment:
Is li + li a typo or does it do something?

Reply:
Of course, it does something. For spacing between those icons, I have used this technique, What it does is. It adds padding in all <li> except first one.

Reply:
@daltonfury42 check now